src/org/thoughtcrime/securesms/conversation/ConversationItem.java

```diff
@@ -797,6 +797,7 @@ private void setLinkPreviewCorners(@NonNull MessageRecord current, @NonNull Opti
   }
 
   private void setContactPhoto(@NonNull Recipient recipient) {
+    if (messageRecord == null) return;
```

Comment:
Do you know under what circumstances this happened?

Reply:
No i was looking into it but i wasn't sure, so i put this guard down just incase but i'm 50% sure this may or may not fix the actual crash.

Reply:
I'll add a note that this is not expected behavior; otherwise it's good to merge